docs/extensions/IconMarkupExtensions.md

```diff
@@ -0,0 +1,126 @@
+</CommandBar>
+```
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Glyph | string | The `string` representing the icon to display. |
+| FontSize | double | The size of the icon to display. |
+| FontFamily | FontFamily | The font family to use to display the icon. If `null`, "Segoe MDL2 Assets" will be used. |
+| FontWeight | FontWeight | The thickness of the icon glyph. |
+| FontStyle | FontStyle | The font style for the icon glyph. |
+| IsTextScaleFactorEnabled | bool | Indicates whether automatic text enlargement, to reflect the system text size setting, is enabled. |
+| MirroredWhenRightToLeft | bool | Indicates whether the icon is mirrored when the flow direction is right to left. |
+
+# FontIconSource markup extension
+The [FontIconSource markup extension](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.extensions.fonticonsourceextension) mirrors the `FontIconExtension` type, but producing `FontIconSource` instances instead of `FontIcon`.
+
+## Syntax
+
+**XAML**
+
+```xml
+<SwipeItems Mode="Reveal">
+
+    <!--Before-->
+    <SwipeItem Text="Accept">
+        <SwipeItem.IconSource>
+            <FontIconSource Glyph="&#xE10B;"/>
+        </SwipeItem.IconSource>
+    </SwipeItem>
+    
+    <!--After-->
+    <SwipeItem Text="Accept" IconSource="{ex:FontIconSource Glyph=&#xE10B;}"/>
+</SwipeItems>
+```
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Glyph | string | The `string` representing the icon to display. |
+| FontSize | double | The size of the icon to display. |
+| FontFamily | FontFamily | The font family to use to display the icon. If `null`, "Segoe MDL2 Assets" will be used. |
+| FontWeight | FontWeight | The thickness of the icon glyph. |
+| FontStyle | FontStyle | The font style for the icon glyph. |
+| IsTextScaleFactorEnabled | bool | Indicates whether automatic text enlargement, to reflect the system text size setting, is enabled. |
+| MirroredWhenRightToLeft | bool | Indicates whether the icon is mirrored when the flow direction is right to left. |
+
+# BitmapIcon markup extension
+The [BitmapIcon markup extension](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.extensions.bitmapiconextension) is similar in structure to the two previous extensions, but it produces `BitmapIcon` instances instead of font-based icons.
+
+## Syntax
+
+**XAML**
+
+```xml
+<MenuFlyout>
+
+    <!--Before-->
+    <MenuFlyoutItem Text="Click me!">
+        <MenuFlyoutItem.Icon>
+            <BitmapIcon Source="/Assets/myicon.png"/>
+        </MenuFlyoutItem.Icon>
+    </MenuFlyoutItem>
+
+    <!--After-->
+    <MenuFlyoutItem
+        Text="No, click me!"
+        Icon="{ex:BitmapIcon Source=/Assets/myicon.png}" />
+</MenuFlyout>
+```
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Source | Uri | The `Uri` representing the image to display. |
+| ShowAsMonochrome | bool | Indicates whether to display the icon as monochrome. |
+
```

Comment:
We should have the `SymbolIcon` too? that the only one missing?

Reply:
Done in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/313/commits/38c86b2e5e046cf2898403dffe72fbcfa7600b45 and https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/313/commits/8e59d8b70bddae188c37e8628048379486f9e188.